<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFindDialog</class>
 <widget class="QDialog" name="CFindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Find &amp;what:</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit</cstring>
   </property>
  </widget>
  <widget class="QCheckBox" name="caseChkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Match &amp;case</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="backwardChkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Search &amp;backward</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="findButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>C&amp;lose</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CFindDialog</receiver>
   <slot>enableFindButton(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findButton</sender>
   <signal>clicked()</signal>
   <receiver>CFindDialog</receiver>
   <slot>findClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>CFindDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableFindButton(QString)</slot>
  <slot>findClicked()</slot>
 </slots>
</ui>
